packages/fast-number-formatter/package.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Please take a look at current master branch - I've moved the config to the root and cleanep up the deps